nix/nixos/cardano-node-service.nix

```diff
@@ -90,9 +106,16 @@ in {
           For details see https://fedoramagazine.org/systemd-template-unit-files/
         '';
       };
+      instances = mkOption {
+        type = types.int;
+        default = 1;
+        description = ''
+          Number of instance of the service to run.
+        '';
+      };
       script = mkOption {
         type = types.str;
-        default = mkScript cfg;
+        default = mkScript cfg 0;
```

Comment:
shouldn't this 0 be cfg.instances?

Reply:
no, `0` is for the first instance, which will be the instance config used by standalone scripts.